ui/app/components/HeaderComponent/HeaderComponent.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Is strict mode enabled by default somehow for all the code? I remember using "use strict"; for setting it up.